### PR TITLE
doc: add example of Digital Ocean Spaces for backup storage

### PR DESCRIPTION
**Pull request by scart88**
Hello everyone, 

This is my first PR and I would like to add an example of Digital Ocean spaces. 
Maybe it will help others as well. 

Thank you and happy holidays! 🎄

### CODE REVIEW

**Review by phisco — APPROVED** [review]

**Comment by mnencia** [comment]
Thank you for your patch. Please include a Signed-off-by line in the commit message as explained in https://github.com/cloudnative-pg/cloudnative-pg/pull/1263/checks?check_run_id=10348455129

**Review by mnencia — APPROVED** [review]

**Comment by sxd** [comment]
Hi @scart88

We can't merge this if you don't sign-off your commit, please find in the error of the CI how you can sign it

Regards,

**Review by armru — APPROVED** [review]

**Pull request closed without merging**
